tests/app/helpers/test_template_helpers.py

```diff
@@ -400,33 +400,80 @@ def test_sign_out_button_text_context(
 
 
 @pytest.mark.parametrize(
-    "survey_config, expected",
+    "survey_config, cookie_present, expected",
     [
-        (SurveyConfig(), f"{ACCOUNT_SERVICE_BASE_URL}/cookies/"),
+        (SurveyConfig(), True, f"{ACCOUNT_SERVICE_BASE_URL}/cookies/"),
         (
             BusinessSurveyConfig(),
+            True,
             f"{ACCOUNT_SERVICE_BASE_URL}/cookies/",
         ),
         (
             NorthernIrelandBusinessSurveyConfig(),
+            True,
             f"{ACCOUNT_SERVICE_BASE_URL}/cookies/",
         ),
         (
             SocialSurveyConfig(),
+            True,
             f"{ACCOUNT_SERVICE_BASE_URL_SOCIAL}/cookies/",
         ),
+        (SurveyConfig(), False, None),
     ],
 )
 def test_cookie_settings_url_context(
-    app: Flask, survey_config: SurveyConfig, expected: str
+    app: Flask, survey_config: SurveyConfig, cookie_present: bool, expected: str
 ):
     with app.app_context():
-        result = ContextHelper(
+        if cookie_present:
+            cookie_session["theme"] = "dummy_value"
+        context_helper = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        )
+        result = context_helper.context.get("cookie_settings_url")
+
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    "survey_config, cookie_present, expected",
+    [
+        (SurveyConfig(), True, f"{ACCOUNT_SERVICE_BASE_URL.replace('https://', '')}"),
+        (
+            BusinessSurveyConfig(),
+            True,
+            f"{ACCOUNT_SERVICE_BASE_URL.replace('https://', '')}",
+        ),
+        (
+            NorthernIrelandBusinessSurveyConfig(),
+            True,
+            f"{ACCOUNT_SERVICE_BASE_URL.replace('https://', '')}",
+        ),
+        (
+            SocialSurveyConfig(),
+            True,
+            f"{ACCOUNT_SERVICE_BASE_URL_SOCIAL.replace('https://', '')}",
```

Comment:
Instead of duplicate `.replace('https://', '')` it can be done once during the assert.